app/models/taxonomy.rb

```diff
@@ -210,6 +210,10 @@ def params_objects
     (self.send("#{type.downcase}_parameters".to_sym).authorized(:view_params) + taxonomy_inherited_params_objects.to_a.reverse!).uniq {|param| param.name}
   end
 
+  def notification_recipients_ids
+    user_ids.uniq
```

Comment:
Just realized we ignore nested orgs

Reply:
I think we should find a proper way to do this inside the DB. Might be hard with MySQL.

Reply:
`self.flat_map(&:subtree).flat_map(&:users).uniq` would return what you want